cognitive/src/main/scala/com/microsoft/azure/synapse/ml/cognitive/CognitiveServiceBase.scala

```diff
@@ -263,6 +263,24 @@ trait HasCognitiveServiceInput extends HasURL with HasSubscriptionKey with HasAA
 
   protected def contentType: Row => String = { _ => "application/json" }
 
+  protected def addHeaders(req: HttpRequestBase,
+                           subscriptionKey: Option[String],
+                           aadToken: Option[String],
+                           contentType: String,
+                           addContentType: Boolean = true): Unit = {
```

Comment:
can't contentType just be empty or non-empty or None? do you need the addContentType arg?

Reply:
Ah yes, thanks for reminding :)